src/inputs/Table/Table.js

```diff
@@ -102,8 +102,12 @@ const _moveExtraPropertiesToCellEditorParams = (col) => {
 
 const _formatColumnsData = (columns, dateFormat) =>
   clone(columns).map((col) => {
-    _formatMinMaxDatesInColumns(col, dateFormat);
-    _moveExtraPropertiesToCellEditorParams(col);
+    if (col.children) {
+      col = { ...col, children: _formatColumnsData(col.children, dateFormat) };
```

Comment:
Could be simplified by using instead:
`col.children = _formatColumnsData(col.children, dateFormat);`